jsonnet/components/kube-state-metrics.libsonnet

```diff
@@ -202,6 +231,8 @@ function(params)
                           --metric-denylist=
                           ^kube_secret_labels$,
                           ^kube_.+_annotations$
+                          ^kube_customresource_.+_annotations_info$,
```

Comment:
it doesn't seem to filter annotations metrics?

Reply:
Ah, I should've mentioned this. So, for testing purposes I didn't spin a cluster but used a local `kind` one, with the configuration in this PR. PTAL below for a set of metrics in an OpenShift 4.15 cluster with this patch applied.

Reply:
Ah, deny-listing doesn't work with CRS metrics. I've skipped both of these metric generations and pushed.